### PR TITLE
Helm readme file generator

### PR DESCRIPTION
**Pull request by dsever**
**Description**

Workflow to generate documentation README file from the values file of DD 

Please see example 

**Test results**

Please see the result https://github.com/dsever/django-DefectDojo/blob/helm_readme_file/helm/defectdojo/README.md 

@alles-klar   @Maffooch do you see this as useful, I've borrowed it from  the bitnami labs looks useful to me to standardize helm

### CODE REVIEW

**Comment by Maffooch** [comment]
Seems very useful to me! I have done a helm install before, but I imagine this would make the process much easier for newcomers

File: helm/defectdojo/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+package manager for Kubernetes. Helm Charts help you define, install, and
+upgrade even the most complex Kubernetes application.
+
+For development purposes,
+[minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+and [Helm](https://helm.sh/) can be installed locally by following
+this [guide](https://helm.sh/docs/using_helm/#installing-helm).
+
+## Supported Kubernetes Versions
+The tests cover the deployment on the lastest [kubernetes version](https://kubernetes.io/releases/) and the oldest supported [version from AWS](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#available-versions). The assumption is that version in between do not have significant differences. Current tested versions can looks up in the [github k8s workflow](https://github.com/DefectDojo/django-DefectDojo/blob/master/.github/workflows/k8s-testing.yml).
```
Comment (alles-klar) [inline helm/defectdojo/README.md:13]: I don't think this part is generated? How does it work, that is is inserted here?

File: helm/defectdojo/values.yaml [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
+## @param createMysqlSecret create mysql secret in defectdojo chart, outside of mysql chart
 createMysqlSecret: false
-# create postgresql secret in defectdojo chart, outside of postgresql chart
+## @param createPostgresqlSecret create postgresql secret in defectdojo chart, outside of postgresql chart
 createPostgresqlSecret: false
-# create postgresql-ha secret in defectdojo chart, outside of postgresql-ha chart
+## @param createPostgresqlHaSecret create postgresql-ha secret in defectdojo chart, outside of postgresql-ha chart
 createPostgresqlHaSecret: false
-# create postgresql-ha-pgpool secret in defectdojo chart, outside of postgresql-ha chart
+## @param createPostgresqlHaPgpoolSecret create postgresql-ha-pgpool secret in defectdojo chart, outside of postgresql-ha chart
```
Comment (alles-klar) [inline helm/defectdojo/values.yaml:15]: We need some documentation how to write this annotations.

File: helm/defectdojo/README.md (hunk 1 above)
Comment (alles-klar) [reply helm/defectdojo/README.md:13]: Maybe the config file is missing in this PR?

**Comment by alles-klar** [comment]
@dsever great one! I think it is very useful.

File: helm/defectdojo/values.yaml (hunk 2 above)
Comment (dsever) [reply helm/defectdojo/values.yaml:15]: Sure https://github.com/bitnami-labs/readme-generator-for-helm

Comment (alles-klar) [reply helm/defectdojo/values.yaml:15]: I am not sure where the best place is, but we need to add this link with one or two lines of description in the values.yaml or in our docs.

**Comment by Maffooch** [comment]
Closing due to stale. Please open again if you'd like to pick this one up again :)

**Pull request closed without merging**